lib/features/context-pads/ContextPads.js

```diff
@@ -117,23 +123,33 @@ ContextPads.prototype.openContextPad = function(element) {
     return;
   }
 
-  var contextPad = this.handlers[element.type].createContextPad(element);
-  
-  if (!contextPad) {
-    return;
-  }
-
-  var position = { top: OFFSET_TOP, left: OFFSET_LEFT };
-
-  var overlayId = this._overlays.add(element, 'context-menu', {
-    position: position,
-    html: contextPad,
-    show: {
-      minZoom: 0.5
+  var contextPads = [];

```

Comment:
I don't understand why the support for multiple context pads is needed. There's no instance of an element having more than one context pad.